dev/wddr/device.c

```diff
@@ -587,6 +591,65 @@ static void wddr_iocal_calibrate(void *dev)
     zqcal_calibrate(&wddr->cmn.zqcal, &wddr->table->cal.common.common.zqcal);
 }
 
+static void ca_enable_loopback(ca_path_t *ca_path, driver_cfg_t *cfg)
+{
+    driver_override_all_bits(&ca_path->tx.ca.driver, WDDR_SLICE_TYPE_CA, cfg, false);
```

Comment:
Just as a curiousity, it is correct that override is false right?

Reply:
Yes, it is. But the variable isn't correct. It should be output enable. I will update.